DependencyInjection/Compiler/FilterPass.php

```diff
@@ -35,7 +35,7 @@ public function process(ContainerBuilder $container)
             
             foreach ($container->findTaggedServiceIds('ongr_filter_manager.filter') as $filterId => $filterTags) {
                 foreach ($filterTags as $tag) {
-                    if (!array_key_exists('manager', $tag)
+                    if (array_key_exists('manager', $tag)
                         && $managerId != ONGRFilterManagerExtension::getFilterServiceId($tag['manager'])
```

Comment:
Something is wrong here: $managerId format is ``ongr_filter_manager.{$filters}`` and getFilterServiceId returns ``ongr_filter_manager.filter.{$filters}``.
Previously tests passed because all filters in tests have manager but now this expression always evaluates to true and everything is skipped.